### ISSUE TITLE
Duplicate text getting pasted in two blocks

### ISSUE THREAD

**Issue by kavir10** [briansunter/logseq-plugin-gpt3-openai#98]
**Issue description:** 
When I am using either gpt or gpt-block commands it is creating two blocks with duplicate content 

**Steps to reproduce:** 
1. Select any text, press Cmd+G and write the prompt
2. Two blocks of duplicated content is visible

1. Press Cmd+J in one block 
2. Two blocks of duplicated content is visible

**Expected behavior:** 
Add only one block of content

**Actual behavior:** 
Creates two blocks of duplicate content

**Environment:** 
Updated the plugin to v1.9.0

**Frequency and severity:**
Each time when used and severity high

**Comment by kavir10** [comment]
I think it resolved when I shut Logseq and started again. But when I disable and enable the plugin, I get this error

<img width="409" alt="image" src="https://user-images.githubusercontent.com/25251144/226114920-e93cff65-c674-4bf1-b2f1-89fb43444099.png">

**Comment by briansunter** [comment]
Looks like this happens on upgrading an existing version. It should go away if you reload logseq. Looking into how to fix it.

**Comment by kavir10** [comment]
Yeah it's not happening anymore after a few days, so would deprioritize this unless it is happening with others too
